app/assets/scripts/components/aa-table-index.js

```diff
@@ -12,9 +12,8 @@ import { setAdmin } from '../actions/action-creators';
 
 const displayHeader = [
   {key: 'name', value: 'Province'},
-  {key: 'done', value: 'Done'},
+  {key: 'field', value: 'Field'},
```

Comment:
`Field` is confusing nomenclature because it already has a meaning in programming, but it'll do for now! Don't be afraid to use more descriptive names, like `fieldData` next time.